content/en/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -15,16 +15,26 @@ This page is an overview of Kubernetes.
 {{% /capture %}}
 
 {{% capture body %}}
-Kubernetes is a portable, extensible open-source platform for managing
-containerized workloads and services, that facilitates both
-declarative configuration and automation. It has a large, rapidly
-growing ecosystem. Kubernetes services, support, and tools are widely available.
+Kubernetes is a cloud infrastructure that provides for the deployment and
+orchestration of containerized applications.    
+It is a portable and extensible open-source platform that is capable of 
+managing both containerized workloads and services. Kubernetes
+provides key features such as a declarative configuration and deployment model,
+built in support for application replicas and autoscaling of applications, and
+also provides support for deploying multiple versions of an application.  
+In addition, Kubernetes has built in health-checking support for its applications,
```

Comment:
This sentence starts “In addition” and ends “as well”. I think just one of those would be enough to make the point.

Reply:
Will fix in the next commit